packages/devtools_server/lib/src/file_system.dart

```diff
@@ -3,24 +3,61 @@ import 'dart:io';
 
 import 'package:path/path.dart' as path;
 
+import 'usage.dart';
+
 class LocalFileSystem {
-  static String userHomeDir() {
+  static String _userHomeDir() {
     final String envKey =
         Platform.operatingSystem == 'windows' ? 'APPDATA' : 'HOME';
     final String value = Platform.environment[envKey];
     return value == null ? '.' : value;
   }
 
-  static File fileFromPath(String pathFromHomeDir) {
-    final file = File('${userHomeDir()}/$pathFromHomeDir');
+  /// Returns the path to the DevTools storage directory.
+  static String devToolsDir() {
+    return path.join(_userHomeDir(), '.devtools');
```

Comment:
oh shoot. For safety we should pick a slightly different name (e,g. 
.dartdevtools for the directory.
otherwise we will cause problems if someone later opens an older version of devtools that assumes .devtools is a file would probably fail. Having the maybeMigrate method is still good as we want that to save your old opt-in setting.

Reply:
done